Repositories/SenderRepository.cs

```diff
@@ -1,32 +1,34 @@
-﻿using shipping_service.Persistence.Entities;
+﻿using shipping_service.Persistence.Database;
+using shipping_service.Persistence.Entities;
 
 namespace shipping_service.Repositories
 {
     public class SenderRepository : ISenderRepository
     {
-        public async Task<List<Sender>> GetAsync()
-        {
-            return new List<Sender>();
-        }
+        private readonly DatabaseContext context;
 
-        public async Task<Sender> GetAsync(ulong id)
+        public SenderRepository(DatabaseContext ctx)
         {
-            return new Sender();
+            context = ctx;
         }
 
-        public async Task<Sender> CreateAsync(Sender sender)
+        public IQueryable<Sender> Senders => context.Senders;
+
+        public async Task CreateAsync(Sender sender)
         {
-            return new Sender();
+            await context.AddAsync(sender);
+            await context.SaveChangesAsync();
         }
 
-        public async Task<Sender> UpdateAsync(ulong id, Sender senderIn)
+        public async Task UpdateAsync(Sender sender)
         {
-            return new Sender();
+            await context.SaveChangesAsync();
         }
 
-        public async Task<ulong> DeleteAsync(ulong id)
+        public void DeleteAsync(Sender sender)
```

Comment:
This should be either be made async, or remove Async in the name for less confusion.